include/samplog/LogLevel.h

```diff
@@ -14,8 +14,15 @@ enum samplog_LogLevel
 	INFO = 2,
 	WARNING = 4,
 	ERROR = 8,
+	FATAL = 16,
```

Comment:
> a purely logging framework should not be so opinionated on "you should not have fatal errors, so we will not record them if you do".  

Well, you have a point there.

But I still feel like a fatal error should be way more noticeable than any other log message. I'll see if I can find a good solution for that. Alternatively there'll be just a general `fatal.log` file.